### PR TITLE
Resolves #8, #9, #10, #14

### LINKED ISSUE #8: Copy highlighted text
- Be able to copy highlighted text and save it to a string (6 hours)

### PR DESCRIPTION
**Pull request by pmdwight**
See commit message. Can delete other folks' sprint1 branches.

### CODE REVIEW

**Review by cyrokamogawa — APPROVED** [review]
Looks good Payton

**Review by cyrokamogawa — COMMENTED** [review]
I wanna let Sean look at this one before I approve cause of the Form1 to ModeForm stuff

File: HotKey/HotKey_MainFolder/HotKeyItem.cs [hunk 1]
```diff
@@ -9,26 +9,84 @@ namespace HotKey_MainFolder
...
+        private Action action;
         private string actionName;
         private ModKeys modKeys;
-        private Keys keys;
+        private Keys key;
 
-        public HotKeyItem() { }
-
-        public HotKeyItem(string actionName, ModKeys modKeys, Keys keys)
+        public HotKeyItem(KeyboardHook hook, Dictionary<Tuple<ModKeys, Keys>, Action> modeFormKeybindDictionary, Action action, string actionName, ModKeys modKeys, Keys key)
```
Comment (ghost) [inline HotKey/HotKey_MainFolder/HotKeyItem.cs]: Just my own personal preference is when something gets this long is to move it down to the next line.

**Review by ghost — APPROVED** [review]
Looks great man. Solid work.

**Pull request merged**